src/picotorrent/ui/widgets/pieceprogressbar.cpp

```diff
@@ -46,7 +48,7 @@ void PieceProgressBar::RenderProgress(wxDC& dc)
         wxMemoryDC memDC;
 
         memDC.SelectObject(prg);
-        memDC.SetBrush(*wxWHITE);
+        memDC.SetBrush(getBGCol());
```

Comment:
We should be able to pass a simple bool to this progress bar saying if it's in dark mode or not. Perhaps even check that bool in the constructor and store the correct brush in the class to avoid doing this lookup in a render function.